hkmc2/shared/src/test/mlscript/backlog/Lifter.mls

```diff
@@ -108,6 +108,9 @@ fun foo() =
 :todo
 :expect "NN"
 foo(0, 2)
```

Comment:
Looks like this was meant to call the other `foo` defined above. We should rename it to avoid the name clash.

Reply:
I see. But don't we have (or plan to do) function overloading?

Reply:
We'll probably have this sort of arity-based overloading later, but it won't work when the several variants are defined in different diff-test blocks. Anyway, testing overloading is clearly not the intent of this test case.